WalletWasabi/WebClients/Wasabi/WasabiClientFactory.cs

```diff
@@ -103,4 +103,4 @@ public void Dispose()
 			GC.SuppressFinalize(this);
 		}
 	}
-}
+}
```

Comment:
Please change your settings, use codemain config file in the repo if needed. 

![image](https://user-images.githubusercontent.com/9844978/106904284-4d9ea880-66fb-11eb-84ee-d76ae5357f22.png)

Reply:
WW files are supposed to end with trailing blank line, right?